internal/docstore/firedocstore/codec_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firedocstore
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"gocloud.dev/internal/docstore/driver"
+)
+
+// TODO(jba): Make this part of the conformance tests. (This differs from the current
```

Comment:
Done.

Reply:
It looks like you deleted the test file entirely, shouldn't it call `RunConformanceTests`?

Reply:
That's going to happen in `fs_test.go`, forthcoming.

Maybe it should define a `CodecTester`, but that's hard because I have to write a server to capture the protos, so I'd rather leave that for a later PR.